cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/task/PollInstanceConsoleOutputTask.java

```diff
@@ -33,4 +34,9 @@ public InstanceConsoleOutputResult call() throws Exception {
     public boolean completed(InstanceConsoleOutputResult instanceConsoleOutputResult) {
         return instanceConsoleOutputResult.getConsoleOutput().contains(CB_FINGERPRINT_END);
     }
+
+    @Override
+    public CloudContext getCloudContext() {
+        return authenticatedContext.getCloudContext();
+    }
```

Comment:
You can use an abstract common class instead of copy-paste this method into all FetchTask implementations.